cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/centrality/AbstractCentrality.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2014 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.algorithms.centrality
+
+import com.twitter.cassovary.graph.DirectedGraph
+
+abstract class AbstractCentrality(graph: DirectedGraph) extends Centrality {
+  val centrality = new Array[Double](graph.maxNodeId + 1)
+
+  /**
+   * Normalize the values
+   * @return
+   */
+  def normalize: Array[Double] = centrality.map(c => c / (graph.maxNodeId - 1))
```

Comment:
From my experience it is not common to divide by node count.  Most often, you will see normalization done by dividing each node's centrality by the maximum number of possible edges (maxNodeId - 1) to other nodes.  

As a reference, here is the documentation for in-degree centrality in python's networkx module:  http://networkx.lanl.gov/reference/generated/networkx.algorithms.centrality.in_degree_centrality.html#networkx.algorithms.centrality.in_degree_centrality